pkg/db/queries.sql

```diff
@@ -95,11 +95,15 @@ FROM
 			address) b ON a.address = b.address
 	AND a.association_sequence_id = b.max_association_sequence_id;
 
--- name: InsertAddressLog :exec
+-- name: InsertAddressLog :execrows
 INSERT INTO address_log(address, inbox_id, association_sequence_id, revocation_sequence_id)
 	VALUES (@address, decode(@inbox_id, 'hex'), @association_sequence_id, @revocation_sequence_id)
-ON CONFLICT
-	DO NOTHING;
+ON CONFLICT (address, inbox_id)
+	DO UPDATE SET
+		revocation_sequence_id = NULL, association_sequence_id = @association_sequence_id
```

Comment:
I wonder if it makes sense to not accept `@revocation_sequence_id` as an argument here, seeing as we are assuming that the caller wants to set it to null?

Reply:
Great point. Think that just got copy/pasted over from the old one